File: dayTwo/python/Level4/Function.py



def add(number1,number2):
    return number1 + number2


print(add(1,2))    


def even (number):
    if number % 2==0:
        print("it is even")
    else:
        print("not even")

even(2)



def square (number):
    return number * number




print(square(2))



def celsius (number):
    fah= (number*9/5) + 32
    return fah




print(celsius(4))




def prime (number):
    isbool=True;

    for index in range(2,number):
      
        if number% index==0:
            isbool= False;
            return "not prime"
            break
          
      
    

   
   


    if isbool==True:

        return "prime"
       





print(prime(13))





def largest(firstNum,secondNum,thirdNum):
    maximum=0


    if firstNum>maximum:

        maximum = firstNum



    if secondNum>maximum:

        maximum = secondNum;




    if thirdNum>maximum:

        maximum = thirdNum;




    return maximum;









print(largest(1,2,3))




def interest(principal,  rate,  time):


    simple= (principal * rate * time)/100;

    return simple;





print(interest(12,12,12))




def area(length,width):
    return length* width


print(area(2,1))

File: dayTwo/javascript/Level4/function.js



function add(number1,number2){

    return number1 + number2
}


console.log(add (1,2))




function even(number){

   if(number%2==0){
     console.log("it is even");
    
}else{
  console.log("it is not even");
}

}
even(2)



function square( number){


   return number*number;

}


console.log(square(2))




function celsius( number){

   

     let fah = (number * 9/5) + 32;

  return fah;




}


console.log(celsius(4))




function rectangle(length, width){

   

     let area = length * width;

  return area;




}

 
console.log(rectangle(4,4)) 





function  prime( number){

   let isbool=true;

   for(let index=2; index<number; index++){
      
    if (number% index==0){
        isbool= false;
      console.log("not prime");
        break
      
      
    }

   
   
}

   if (isbool==true){

  console.log("prime");
   }


}


prime(10)



function interest(principal,  rate,  time){


let simple= (principal * rate * time)/100;

return simple;



}

console.log(interest(12,12,12))







function maximum(firstNum, secondNum,  thirdNum){

    let maximum=0;

    if(firstNum>maximum){

        maximum = firstNum;
    
}

  if(secondNum>maximum){

        maximum = secondNum;
    
}


  if(thirdNum>maximum){

        maximum = thirdNum;
    
}


return maximum;






}




console.log(maximum(1,2,3))



















